Mathlib/RingTheory/Valuation/ValuationSubring.lean

```diff
@@ -643,7 +643,7 @@ theorem ker_unitGroupToResidueFieldUnits :
 theorem surjective_unitGroupToResidueFieldUnits :
     Function.Surjective A.unitGroupToResidueFieldUnits :=
   (IsLocalRing.surjective_units_map_of_local_ringHom _ Ideal.Quotient.mk_surjective
-        IsLocalRing.isLocalHom_residue).comp
+  (inferInstanceAs (IsLocalHom (IsLocalRing.residue A)))).comp
     (MulEquiv.surjective _)
```

Comment:
I find the indentation, combined with the number of parens, slightly confusing. Perhaps
```lean
  IsLocalRing.surjective_units_map_of_local_ringHom _ Ideal.Quotient.mk_surjective
    (inferInstanceAs (IsLocalHom (IsLocalRing.residue A))) |>.comp (MulEquiv.surjective _)
```
?